app/src/assets/localization/en/protocol_setup.json

```
@@ -6,31 +6,26 @@
   "extra_attention_warning_title": "Secure labware and modules before proceeding to run",
   "labware_setup_step_description": "Position full tip racks and labware in the deck slots as shown in the deck map.",
   "labware_setup_step_title": "Labware Setup",
-  "labware_help_link_title": "See How Labware Offsets Work",
-  "how_offset_data_works_title": "How Labware Offsets Work",
-  "position_offset_overview_and_description": "<h4>Positional Adjustments Overview</h4><block>The OT-2 provides two types of positional adjustments. The first type is robot calibration: Deck calibration, Tip Length calibration and Pipette Offset calibration. These calibrations ensure the OT-2 moves to the expected positions on the deck. It is essential to have good robot calibration before creating any labware offsets and running protocols.</block>",
-  "learn_more_about_robot_cal_offset_modal_link": "Learn more about Robot Calibration",
-  "creating_labware_offset_data": "<h4>Creating Labware Offset Data During Labware Position Check</h4><block>Labware Position Check is a recommended workflow that helps you verify the position of every labware on the deck for an added degree of precision in your protocol. When you check a labware, the OT-2’s pipette nozzle or attached tip will stop at the center of the A1 well. If the pipette nozzle or tip is not centered, you can reveal the OT-2’s jog controls to make an adjustment. This Labware Offset will be applied to the entire labware. Offset data is measured to the nearest 1/10th mm and can be made in the X, Y and/or Z directions.</block>",
-  "labware_offset_data_info": "Opentrons offers a second type of positional adjustment, a Labware Offset. Labware Offsets are intended to account for small, real-world variances in the overall position of the labware on an OT-2’s deck. A Labware Offset is unique to a specific combination of the following during a run: labware, module (if present), deck slot, protocol and OT-2. ",
-  "learn_more_about_offset_data_link": "Learn more about Labware Offset Data",
+  "how_offset_data_works": "How labware offsets work",
```

nit maybe? but totally optional.
```suggestion
  "how_labware_offsets_work": "How labware offsets work",
```